CI/install-dependencies-linux-ubuntu16.sh

```diff
@@ -55,5 +55,14 @@ cmake -DENABLE_TESTING=Off -DUSE_SHARED_MBEDTLS_LIBRARY=On ../mbedtls-mbedtls-2.
 make -j 12
 sudo make install
 
+# build cef
+cd ~/projects
+curl -L "http://opensource.spotify.com/cefbuilds/cef_binary_76.1.5%2Bgd8a577c%2Bchromium-76.0.3809.87_linux64_minimal.tar.bz2" -o "cef.tar.bz2"
```

Comment:
We haven't tested our end with 3809. We're likely downgrading from 3770 (which Windows has in v24) to **3729**, so I'd use that.

Reply:
for cherry picking in the changes on bazukas branch here is what i did
```
cd ./plugins/obs-browser
git remote add bazuka https://github.com/bazukas/obs-browser.git
git fetch bazuka
git cherry-pick f536b1457009607a47a15bf02ef14010b647cf81
```

Reply:
Its been changed to 3770 now.